FirebaseStorage/Sources/Internal/StorageGetDownloadURLTask.swift

```diff
@@ -92,15 +92,15 @@ internal class StorageGetDownloadURLTask: StorageTask, StorageTaskManagement {
     }
   }
 
-  internal func downloadURLFromMetadataDictionary(_ dictionary: [String: String]) -> URL? {
+  internal func downloadURLFromMetadataDictionary(_ dictionary: [String: Any]) -> URL? {
     let downloadTokens = dictionary["downloadTokens"]
-    guard let downloadTokens = downloadTokens,
+    guard let downloadTokens = downloadTokens as? String,
```

Comment:
Just a review note that my Swift-skills are terrible, so I'm not 100% sure that the fallback of else/return nil will be hit if the type coercion of `as?` fails. Same below - I *think* the fallback is hit if the coercion fails I'm just not sure exactly of order of operations between guard/as?/else etc

Reply:
It is.